packages/css/tests/index.test.ts

```diff
@@ -995,4 +995,48 @@ describe("createCss", () => {
       ._bZKhEt{font-size:1.2em;}"
     `);
   });
+
+  test("should not yield type errors when raw values are used even though tokens are defined", () => {
+    const css = createCss({
+      tokens: {
+        colors: {
+          primary: "red",
+        },
+      },
+    });
+    css({
+      color: "lime",
+    });
+  });
+
+  test("should yield type errors in strict mode when tokens are defined but raw values are used", () => {
+    const css = createCss({
+      strict: true,
+      tokens: {
+        colors: {
+          primary: "red",
+        },
+      },
+    });
+    css({
+      color: "primary",
+      // @ts-ignore
```

Comment:
I saw in a diff PR you used `@ts-expect-error`. just wondering if its also worth using it here instead of a `@ts-ignore`?